### PR TITLE
DOC: Add example to MoreauEnvelope documentation

### PR DESCRIPTION
**Pull request by adler-j**
I felt this example grealy clarifies what the envelope does.

### CODE REVIEW

**Comment by adler-j** [comment]
Trivial, short review?

File: odl/solvers/functional/default_functionals.py [hunk 1]
```diff
@@ -2135,6 +2135,17 @@ class MoreauEnvelope(Functional):
         [\\nabla \mathrm{env}_{\\sigma  f}](x) =
         \\frac{1}{\\sigma} (x - \mathrm{prox}_{\\sigma  f}(x))
 
+    Example, if :math:`f = \| \cdot \|_1`, then
```
Comment (kohr-h) [inline odl/solvers/functional/default_functionals.py]: Example: If

File: odl/solvers/functional/default_functionals.py [hunk 2]
```diff
@@ -2135,6 +2135,17 @@ class MoreauEnvelope(Functional):
...
+
+    .. math::
+        [\mathrm{env}_{\\sigma  \| \cdot \|}(x)]_i =
```
Comment (kohr-h) [inline odl/solvers/functional/default_functionals.py]: subscript 1 missing

File: odl/solvers/functional/default_functionals.py [hunk 3]
```diff
@@ -2135,6 +2135,17 @@ class MoreauEnvelope(Functional):
...
+        \\begin{cases}
+            \\frac{1}{2 \\sigma} x_i^2 & \\text{if } |x_i| \leq \\sigma \\\\
+            |x_i| - \\frac{\\sigma}{2} & \\text{if } |x_i| > \\sigma
```
Comment (kohr-h) [inline odl/solvers/functional/default_functionals.py]: comma after sigma

**Comment by adler-j** [comment]
Merge after CI.

**Comment by adler-j** [comment]
This "This branch is out-of-date with the base branch" combined with having to wait for travis is getting out of hand, I've rebased this 3 times now :/

**Comment by kohr-h** [comment]
> This "This branch is out-of-date with the base branch" combined with having to wait for travis is getting out of hand, I've rebased this 3 times now :/

Yeah, what can you do when there's a merge every 10 minutes..

Perhaps it makes sense to switch off the CI requirement in the settings. But there will be calmer days again.

**Pull request merged**